baconclient/baconclient.go

```diff
@@ -113,24 +112,25 @@ func (b *BaconClient) AddRpc(rpcId int, rpcEndpointUrl string) {
 	b.rpcClients = append(b.rpcClients, newBaconSlice)
 
 	// Launch client
-	waitGroup.Add(1)
+	b.waitGroup.Add(1)
 	go b.blockWatch(newBaconSlice)
 }
 
 func (b *BaconClient) Shutdown() {
 	b.Signer.Close()
 }
 
-func (b *BaconClient) ShutdownRpc(rpcId int) error {
+func (b *BaconClient) ShutdownRPC(rpcID int) error {
 
-	newClients := make([]*BaconSlice, 0)
+	var newClients []*BaconSlice
```

Comment:
What's the difference here? I thought in go you should always use make() to init your slices?

Reply:
mostly a stylistic thing https://github.com/uber-go/guide/blob/master/style.md#use-var-for-zero-value-structs, make maps, var for all else unless the size is known